opengrok-indexer/src/main/java/org/opengrok/indexer/framework/PluginClassLoader.java

```diff
@@ -257,6 +255,8 @@ public Class loadClass(String name, boolean resolveIt) throws ClassNotFoundExcep
             }
         }
 
+        checkClassname(name);
```

Comment:
this does not prevent loading classes which are already loaded by the parent loader